packages/react/src/components/TextInput/TextInput.tsx

```diff
@@ -6,7 +6,7 @@
  */
 
 import PropTypes from 'prop-types';
-import React, { useContext, useState } from 'react';
+import React, { ReactNode, useContext, useState } from 'react';
 import classNames from 'classnames';
 import { useNormalizedInputProps } from '../../internal/useNormalizedInputProps';
 import PasswordInput from './PasswordInput';
```

Comment:
`import * as FeatureFlags from '@carbon/feature-flags';` gives me an error saying that the module is not found but this line is unchanged from when i converted it to .tsx, thought I would bring it up to get comments on that?

Reply:
not related to code, did not run `npm run build` at that time